build.rs

```diff
@@ -28,6 +28,8 @@ fn main() -> Result<()> {
         }
     };
 
+    let target = std::env::var("TARGET").expect("The 'TARGET' environment variable MUST be set");
```

Comment:
TARGET RUNTIME

Reply:
`TARGET` is built-in predefined (rust) env variable here (always exists) we can't unfortunately control